src/i18n/en_GB.ts

```diff
@@ -629,12 +629,12 @@ export default {
             coins: 50,
             credits: 1_000_000,
             extensions: [],
-            levelcost: [],
+            levelcost: [1.000.000 / 50 Coins],
```

Comment:
need single quotes `'` around the string: `['1.000.000 / 50 Coins']`

Reply:
OK I'll fix that